### DISCUSSION TITLE
Crash when activating Follow mode without GPS on

### DISCUSSION THREAD

**Discussion by JimmyS83** [devemux86/cruiser#284] [Q&A] [answered]
Hi,

It must be introduced with some of recent versions, but I found, when GPS isnt on, and I long press GPS button to activate follow mode, aplication crashes. Route was imported, not sure if thats factor or not.

**Comment by MotoUKRider** [comment]
Confirmed here, too.  Application exits with "Location is uknown" - also same on initial load with no route loaded.
(Samsung S20, Android 13)

**Comment by devemux86** [comment]
@JimmyS83 @MotoUKRider 

Thanks for the report!
It will be fixed in next Cruiser.

**Comment by devemux86** [comment, marked as answer]
Fixed in [Cruiser 3.0.31](https://play.google.com/store/apps/details?id=gr.talent.cruiser).
